### PR TITLE
Query used sgqlc schema update

### PR DESCRIPTION
**Pull request by ddjnw1yu**
These updates will be used in the details page:
- separate sgqlc schema
- error handling
- api testing

### CODE REVIEW

File: app/sgqlc.py [hunk 1]
```diff
@@ -38,87 +51,104 @@ def convert_query(self, item, query):
...
             return self.convert_query(
                 item,
-                query.datasetDescription(
-                    first=item.limit,
-                    offset=(item.page-1)*item.limit,
-                    submitter_id=item.filter["submitter_id"] if "submitter_id" in item.filter else None
+                query.manifestFilter(
+                    first=0,
+                    offset=0,
+                    additional_types=item.filter["additional_types"] if "additional_types" in item.filter else None
```
Comment (alan-wu) [inline app/sgqlc.py]: A get function for dictionary will look better - 
e.g. `additional_types=item.filter.get("additional_types", None)`

File: app/sgqlc.py [hunk 2]
```diff
@@ -38,87 +51,104 @@ def convert_query(self, item, query):
...
                 item,
-                query.datasetDescriptionFilter(
-                    first=item.limit,
-                    offset=(item.page-1)*item.limit,
+                query.caseFilter(
+                    first=0,
+                    offset=0,
+                    species=item.filter["species"] if "species" in item.filter else None,
+                    sex=item.filter["sex"] if "sex" in item.filter else None,
+                    age_category=item.filter["age_category"] if "age_category" in item.filter else None
```
Comment (alan-wu) [inline app/sgqlc.py]: Same as above.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good, I have a small suggestion on one of the syntax.

File: app/sgqlc.py (hunk 1 above)
Comment (ddjnw1yu) [reply app/sgqlc.py]: Thanks, will update it now.

**Pull request merged**